templates/newslist.html

```diff
@@ -48,31 +48,16 @@ <h2>CoffeeScript</h2>
   <p>An unprecedented case of joviality and camaraderie has struck HackSoc, and we have responded with a weekly social over in the Glasshouse on Hes East. Come for lunch, a chat, and maybe a small game or two.</p>
 </div>
 
-<div class="infobox">
-  <h2>Game Nights</h2>
-  <p>It's reasonably safe to say we are a society with technology, and what's technology good for other than programming? Gaming! We'll hook up some consoles to projectors and fritter the night away.</p>
-</div>
-
 <div class="infobox alternate">
   <h2>HackPub</h2>
-  <p>Another tradition with a long and glorious history, the humble HackPub, in which we visit one of York's many fine pubs and converse late into the night about high culture, politics, and how old jet engines work over a pint (of something non-alcoholic, if you so desire).</p>
-</div>
-
-<div class="infobox">
-  <h2>HackTogether</h2>
-  <p>A new addition to our roster, gathering in a computer lab together to work on projects either individually or together, and ordering pizza. Being alone together can be a lot more fun than being alone by yourself.</p>
+  <p>Another tradition with a long and glorious history, the humble HackPub, in which we visit the Rook and Gaskill at Walmgate and converse late into the night with members <b>past<\b> and present about high culture, politics, and how old jet engines work over a pint (of something non-alcoholic, if you so desire).</p>
```

Comment:
"at Walmgate" - "at", and it's on Lawrence Street

Also, backslash

Reply:
And it's not *entirely* set that it's at R&G

Reply:
What should "at Walmgate" be changed to?

Reply:
@LordAro #58 tried to fix, *new* committee today decided that there needed to be a location and also a little more emphasis on the amount of alumni, postgrads, (non-freshers basically) there :P